app/code/Magento/InventoryInStorePickupApi/README.md

```diff
@@ -0,0 +1,25 @@
+# InventoryInStorePickupApi module
+
+The `InventoryInStorePickupApi` module provides Inventory Management service contracts. 
+
+This module is part of the new inventory infrastructure. The
+[Inventory Management overview](https://devdocs.magento.com/guides/v2.3/inventory/index.html)
+describes the MSI (Multi-Source Inventory) project in more detail.
+
+## Installation details
+
+This module is installed as part of Magento Open Source. Unless a custom implementation for `InventoryInStorePickupApi`
+is provided by a 3rd-party module, the module cannot be deleted or disabled.
+
+## Extensibility
+
+The `InventoryInStorePickupApi` module contains extension points and APIs that 3rd-party developers
+can use to provide custom inventory source selection algorithm functionality.
```

Comment:
> to provide custom inventory source selection algorithm functionality.

instead of " custom inventory source selection algorithm functionality."

it should be something like - customization of In Store Pickup functionality